lib/Doctrine/DBAL/Internal/CommitOrderEdge.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Doctrine\DBAL\Internal;
+
+class CommitOrderEdge
```

Comment:
Could these two classes be final?

Reply:
They could, but as they are created within the calculator it doesn't make a difference.